src/main/java/webserver/request/HttpRequest.java

```diff
@@ -90,4 +83,10 @@ public String getParameter(String key) {
 
         return param;
     }
+
+    public String getSessionId() {
+        return Cookies.of(getHeader(HttpHeader.COOKIE))
+            .getSessionId();
+    }
```

Comment:
```suggestion
    public String getSessionId() {
        return this.cookies.get(HttpHeader.JSESSIONID);
    }
```
HttpReqeust 가 Cookies 객체를 가지고 있었다면 위와 같이 간단히 표현 가능했을것 같아요. 
쿠키 값을 얻기 위해 매번 Cookies.of() 와 같이 객체 생성을 하지 않아도 될것 같구요 ㅎㅎ

Reply:
아.. 만들면서도 쿠키를 Request 에 필드로 넣을까말까 필드가 너무 많지 않은가 했었는데요 ㅎㅎ;
필드로 넣는다면 그래도 쿠키는 Header 보다는 Request 에 넣어보겠습니다!!